keytree/src/lib.rs

```diff
@@ -79,6 +79,22 @@ impl Xprv {
         }
     }
 
+    /// Returns a leaf Xprv. (Is this really an xprv, or just a prv?)
```

Comment:
One question here. Are these leaf keys really _extended_ keys, or just regular ole keys?

Reply:
Leaf secret is Scalar. Leaf pubkey is CompressedRistretto (we’ll change it later to VerificationKey)

Reply:
Ohhh. Oops!

Reply:
Ok. Does that mean I should just refer to this as "leaf secret" and "leaf pubkey"?

Reply:
Maybe “leaf private/public key”? If api is simply called `derive_key` then you can talk about private/public _keys_ and extended private/public keys.